errors/parking_bay.yaml

```diff
@@ -0,0 +1,6 @@
+description: >-
```

Comment:
You could just point to generic.yaml for errors if there are no specific errors yet. This will also include VendorError/Warning which should be available at each HW driver interface to raise HW vendor specific errors.